Formula/git-game.rb

```diff
@@ -1,8 +1,8 @@
 class GitGame < Formula
   desc "Game for git to guess who made which commit"
   homepage "https://github.com/jsomers/git-game"
-  url "https://github.com/jsomers/git-game/archive/1.2.tar.gz"
-  sha256 "d893b2c813388754c16d867cf37726cd7e73c9ccd316735aac43bf1cd3ab1412"
+  url "https://github.com/coffee-dan/git-game/archive/1.3.tar.gz"
```

Comment:
We should probably check with @jsomers that they're happy for you to take over the project before switching this to your repo.

Reply:
Absolutely! Thank you for pointing this out. 

I setup this PR to get ahead of CI failures. I am coordinating with @jsomers to make a 1.3 release on the official git-game repo and _that_ will be the URL for that gets used.